shared/lib_pv_io_manager.cpp

```diff
@@ -491,7 +491,8 @@ void PVSystem_IO::SetupPOAInput()
 				double sun[9], angle[5];
 				int tms[3];
 
-				solarpos(wf.year, wf.month, wf.day, 12, 0.0, hdr.lat, hdr.lon, hdr.tz, sun);
+				//solarpos(wf.year, wf.month, wf.day, 12, 0.0, hdr.lat, hdr.lon, hdr.tz, sun);
+                solarpos_spa(wf.year, wf.month, wf.day, 12, 0.0, 0.0, hdr.lat, hdr.lon, hdr.tz, 0, 66.7, hdr.elev, wf.pres, wf.tdry, 0, 180, sun);
```

Comment:
Here, `wf.tdry` is a time series of temperature, right? The SPA algorithm actually specifies annual average temperature (and pressure) for the refraction calculation. If SAM is adopting SPA it would be good to have pvlib-python be consistent with SAM's implementation. The [issue](https://github.com/pvlib/pvlib-python/issues/1065#issuecomment-701616850) of temperature and pressure for pvlib-python's spa code came up recently.

Reply:
Hi Cliff, 

Yes SAM is planning to adopting SPA. That is a great point on the time series vs. annual average, and I would tend to agree that SAM would like to be consistent with pvlib's inputs. Adding more experienced members of the SAM team @dguittet @sjanzou @cpaulgilman to provide input as needed.

Reply:
@mjprilliman we're in agreement on consistency, but now waiting for someone to decide first :) The pvlib maintainers are hesitant to use time-dependent temperature because that's a departure from the algorithm as published.

Reply:
Hmm I'd be inclined to follow your lead on that. I haven't worked with the SPA long enough to have a strong opinion on it either way. If I'm following the comment chain on the pvlib issue correctly, pvlib is currently using time-dependent temperatures? I don't think it would be an issue to revise my code to use the annual average instead if that were the preferred course of action. I'm not sure if Janine @janinefreeman would have a strong opinion either way. @dguittet thoughts?

Reply:
> pvlib is currently using time-dependent temperatures?

Yes, which was done without recognizing that is a departure from the publication.

My opinion is that either time-dependent or annual average temperature and pressure could be used, based on a brief email exchange with Ibrahim. The downside of using annual average is that not every use case will have a full year of weather data, which requires a user to bring additional data.

Reply:
I think if sticking with the time-dependent temperature and pressure doesn't violate any part of the original publication, it's a defensible departure.

Reply:
Can you clarify what you would consider a violation? Using time-dependent temperature may not be a technically wrong, but the original publication clearly states annual average temperature.

Reply:
Yes, sorry, I forgot the word "assumption". I wonder that if the original authors' assumption is "the required temperature and pressure can't be known and the annual average is a good trade-off between correcting for +/- differences within all the timesteps and incorporating local information" then it seems adding those +/- differences by using time-dependent values may either balance out downstream of the simulation, or not amount to much, or perhaps adds in something that tilts the trade-off. Instead, if it's meant to provide consistency, why not use a decade average temperature? So without knowing why the annual average is used in lieu of other possibilities, and reading of your discussion with the author, I just presumed it wasn't violating some important assumption.

I definitely agree with the move towards model alignment. Since @janinefreeman 's original interest was to align with NSRDB, I think it'll be worth meeting with them to learn their take / implementation.

Reply:
I think given Aron's agreement with @cwhanse that the results would be similar, equally valid, and have minimal differences in uncertainty, and that perhaps it was an issue of what data was readily available, I'm fine continuing to use time-dependent info; for single time step simulations and that both softwares are already doing this, as pointed out above.